Dockerfile

```diff
@@ -86,7 +82,19 @@ RUN \
   git clone --depth 1 --branch v$AOM_VERSION "$AOM_URL" && \
   cd aom && test $(git rev-parse HEAD) = $AOM_COMMIT && \
   mkdir build_tmp && cd build_tmp && \
-  cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DENABLE_EXAMPLES=NO -DENABLE_TESTS=NO -DENABLE_TOOLS=NO -DCONFIG_TUNE_VMAF=1 -DENABLE_NASM=ON -DCMAKE_INSTALL_LIBDIR=lib .. && \
+  cmake \
+    -G"Unix Makefiles" \
```

Comment:
Wonder if makefile generator is default on unix systems?

Reply:
Yeah guess so too, but maybe make sense to make it explicit if we use `-DCMAKE_*` defines? have no strong opinions about it